server/py/services/api/initial_data.py

```diff
@@ -993,6 +998,47 @@ def _create_project_summaries(db, db_session):
     db._upsert(db_session, project_summaries, ignore=True)
 
 
+def _init_system_id(db_session: sqlalchemy.orm.Session):
+    """
+    Initializes a system id for MLRun deployment.
+    The system id is first checked in the database. If it does not exist, the function checks for a configured id
+    (from environment variables or config), and if neither is found, a new random one is generated and stored.
+    """
+
+    db = framework.db.sqldb.db.SQLDB()
+
+    # check if a system id already exists in the database
+    system_id = db.get_system_id(db_session)
+
+    if system_id is not None:
+        logger.info("Existing system id found in the database", system_id=system_id)
+    else:
+        # check if the system id is already set in the config
+        system_id = _get_configured_system_id()
+
+        if system_id is not None:
+            logger.info("Using configured system id", system_id=system_id)
+            db.store_system_id(db_session, system_id)
+        else:
+            # if no system id is found, generate a new one
+            system_id = _generate_system_id()
+            db.store_system_id(db_session, system_id)
+
+    # set the system id in mlrun config
+    mlrun.mlconf.system_id = system_id
```

Comment:
```suggestion
    if system_id is not None:
        logger.info("Existing system id found in the database", system_id=system_id)
        mlrun.mlconf.system_id = system_id
        return
  
    # check if the system id is already set in the config
    system_id = _get_configured_system_id()
  
    <Liran suggestion above>
    # set the system id in mlrun config
    mlrun.mlconf.system_id = system_id
```

This way we we improve readability and unnecessary nested structure